src/App.css

```diff
@@ -1,38 +1,117 @@
-.App {
+body {
+  margin: 0%;
```

Comment:
isn't 0 the default value?

Reply:
I had to add the that because otherwise a white border appears